docs/guides/advanced/schemas.md

```diff
@@ -459,6 +459,112 @@ In your sync formula you only need to populate the fields of the reference objec
+});
+```
+
+<img src="../../../images/schemas_labels_default.png" srcset="../../../images/schemas_labels_default_2x.png 2x" class="screenshot" alt="Default labels for subtitle properties">
+
+However, there are times when the default label isn't a great fit. For instance you may want to put the property name after the value (`10 bugs` instead of `Bugs: 10`) or remove the label completely (`P1` instead of `Priority: P1`).
+
+To customize the label, when specifying the `subtitleProperties` pass a [`PropertyIdentifierDetails`][PropertyIdentifierDetails] object instead of a string. Set the `property` field to the path of the property (what the string value normally contains) and set the `label` field to the template string to use for the label.
+
+There are three options for setting the label:
+
+1.  Pass a string which will be used as an alternative label.
+1.  Pass a string containing the constant [`PropertyLabelValueTemplate`][PropertyLabelValueTemplate], which acts as a placeholder for where the property's value should be inserted.
+1.  Pass an empty string, which will remove the label completely.
+
+```ts
+let MovieSchema = coda.makeObjectSchema({
+  // ...
+  subtitleProperties: [
+    "director",
+    // Fully customize the label for the year property.
+    { property: "year", label: `Released in ${coda.PropertyLabelValueTemplate}` },
+    // Only show the value of the rating property.
+    { property: "rating", label: "" },
+  ],
+});
+```
+
+<img src="../../../images/schemas_labels_custom.png" srcset="../../../images/schemas_labels_custom_2x.png 2x" class="screenshot" alt="Custom labels for subtitle properties">
+
+
+## Property paths
+
+Object schema fields that expect a property name (`titleProperty`, `snippetProperty`, etc) also accept a path to a property on a nested object. The paths are specified using a subset of the [JSONPath syntax][jsonpath], which at it's simplest just joins the property names with a dot (like `property1.property2`).
+
+!!! warning "Only supported for card fields"
+    At the moment property paths can only be used in the object [schema fields used by cards](#cards).
+
+Consider the following schema for a movie:
+
+```ts
+let PersonSchema = coda.makeObjectSchema({
+  properties: {
+    name: { type: coda.ValueType.String },
+    // ...
+  },
+});
+
+let MovieSchema = coda.makeObjectSchema({
+  properties: {
+    director: PersonSchema,
+    actors: {
+      type: coda.ValueType.Array,
+      items: PersonSchema,
+    },
+    // ...
+  },
+});
+```
+
+The following property Paths are all valid:
+
+- `director.name` - The name of the director.
+- `actors[0].name` - The name of the first actor.
+- `actors[*].name` - The names of all of the actors (comma separated).
+
+If you need to further customize the value, such as combining the value of multiple properties or doing some other transformation, you'll need to create a new property to hold that value and manually populate it in your `execute` function.
```

Comment:
might be useful to show an example of this too that's more complex, maybe a custom `titleProperty` that combines a few things from an API request, but purely additional

Reply:
I have an example above in the Display value section. I added a link there.